src/main/java/de/doubleslash/keeptime/Main.java

```diff
@@ -260,6 +266,31 @@ public void handle(final WindowEvent event) {
 
    }
 
+   private void registerMinifyEventlistener(final Scene mainScene, final Stage primaryStage) {
+      final KeyCode down = KeyCode.DOWN;
+      final KeyCombination keyComb = new KeyCodeCombination(down, KeyCombination.META_DOWN);
```

Comment:
cool, didn't know this 👍  should also use this in the listener for the popup

Reply:
instead of jnative hook?